### DISCUSSION TITLE
Question about Command Search and filters

### DISCUSSION THREAD

**Discussion by kenfhill84083** [freeplane/freeplane#1173] [Help] [answered]
In Command Search, typing "ancestor" results in "Filter-> Show ancestors" displaying as the only item (only Menu Items is checked).

Typing "descendant" results in "Filter-> Show descendants displaying as the only item.

But typing "filter" does not display anything to do with either ancestors or descendants.

That seems illogical to me.  Just trying to understand.

You can see a short, unlisted video showing this at https://youtube.com/shorts/_UrOetUsxP4?feature=share

**Comment by dpolivaev** [comment]
The menus from the menu bar are excluded from the search because they would generate too many results otherwise.

**Reply by kenfhill84083** [reply]
> The menus from the menu bar are excluded from the search because they would generate too many results otherwise.

Sorry, I don't understand.  Your answer sounds like a blanket statement.  But in the attached image, most of items on the Filter menu (green arrows) show up in Command Search.  Are some tagged with a priority marker that excludes them from CS?

![filter _ menu , CS](https://github.com/freeplane/freeplane/assets/6380301/72789270-b65d-4035-97f5-dde2cbe7b835)

**Comment by kenfhill84083** [comment, marked as answer]
Never mind.  I can see each green-arrowed item has "filter" in the label.
